lib/core/request.js

```diff
@@ -9,6 +9,21 @@ const assert = require('assert')
 
 const kHandler = Symbol('handler')
 
+const channels = {}
+
+try {
+  const diagnosticsChannel = require('diagnostics_channel')
+  channels.create = diagnosticsChannel.channel('undici:request:create')
+  channels.bodySent = diagnosticsChannel.channel('undici:request:bodySent')
```

Comment:
Not documented. Can we skip this for now?

Reply:
I actually forgot to document.